src/main/java/org/zalando/pazuzu/feature/FeaturesResource.java

```diff
@@ -32,10 +32,16 @@ public FeaturesResource(FeatureService featureService) {
     @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
     public List<FeatureDto> listFeatures(
             @RequestParam(required = false, name = "name") String[] featureNames,
-            @RequestParam(required = false, name = "sorted") Integer sorting)
+            @RequestParam(required = false, name = "sorted") Integer sorting,
```

Comment:
As said before, IMO this parameter should be renamed to something like `includeDependencies=true`, `resolveDependencies`, `withDependencies=true`, `dependencies=true`. When getting dependencies you will get them sorted topologically, but the main reason for that param is for dependencies to get included into the result set.